public/index.html

```diff
@@ -1,12 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <!-- OG, TW -->
```

Comment:
What's the purpose of this?

Reply:
What do you mean? It's just a html comment just to add clarity for sections

Reply:
Yes, but what is `TW` :sweat_smile:? Doesn't seem to be clear.